cli/app.go

```diff
@@ -1646,6 +1648,33 @@ Example:
 							},
 							Action: ModuleBuildLogsAction,
 						},
+						{
+							Name:  "link-repo",
+							Usage: "link a GitHub repository to your module",
+							//nolint:lll
+							UsageText: `This command connects a Viam module to a GitHub repository so that repo actions can trigger builds and releases of your module.
+
+This won't work unless you have an existing installation of our GitHub app on your GitHub org. (Details to follow).
+`,
+							Hidden: true,
+							Flags: []cli.Flag{
+								&cli.StringFlag{
+									Name:  moduleBuildFlagOAuthLink,
+									Usage: "ID of the oauth link between your GitHub org and Viam. Only required if you have more than one link",
+								},
+								&cli.StringFlag{
+									Name:     moduleFlagPath,
+									Usage:    "your module's ID in org-id:name or public-namespace:name format",
+									Required: true,
```

Comment:
[opt] infer if `meta.json` exists in CWD. Not sure if it is better to just force people to type it (may be easier to explain in the docs if it is required ? )

(via `loadManifestOrNil`(you added that func iirc) )

Reply:
smart change, it now autodetects module ID and git repo if available in current folder